visionmetrics/caption/azure_openai_model_eval.py

```diff
@@ -0,0 +1,42 @@
+from visionmetrics.caption.azure_openai_model_eval_base import AzureOpenAITextModelCategoricalEvalBase
+
+
+class AzureOpenAITextModelCategoricalScore(AzureOpenAITextModelCategoricalEvalBase):
+    """
+    Prompt-tuned evaluator that uses single-turn text-only calls to an Azure OpenAI model for evaluation and returns
+    standard categorical metrics (precision, recall, F1, accuracy) based on the scores returned.
+    """
+
+    def __init__(self):
+        self.system_message = """You are an expert evaluator of AI models. Follow the steps below:
```

Comment:
Do we expect tuning the prompts in experiments? If so, maybe we can make this system message and prompt as default values, and add an argument to customize it?

Reply:
Maybe define this SYSTEM PROMPT outside the class as a constant - if the user doesn't provide their system prompt, then default gets used as Chongyang suggested.

Reply:
GMTA, was thinking the same thing today. Updated!